Demo/Demo/SceneDelegate.m

```diff
@@ -0,0 +1,59 @@
+#import "SceneDelegate.h"
+#import <PopupBridge/POPPopupBridge.h>
+
+@interface SceneDelegate ()
+
+@end
+
+@implementation SceneDelegate
+
+- (void)scene:(UIScene *)scene openURLContexts:(NSSet<UIOpenURLContext *> *)URLContexts {
+   for (UIOpenURLContext *urlContext in URLContexts) {
+       NSURL *url = [urlContext URL];
+       if ([url.scheme localizedCaseInsensitiveCompare:@"com.braintreepayments.popupbridgeexample"] == NSOrderedSame) {
+       [POPPopupBridge openURL:urlContext.URL];
+       }
+   }
+}
+
+- (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions {
```

Comment:
We can delete all of these empty methods.

Reply:
I left all the empty methods & comments because I think it makes future debugging issues with scenes/multitasking easy since each method has a nice description of what it does.

Reply:
We can remove though if we think leaving them in is confusing.

Reply:
Either way! We can leave them if you find them helpful.